docs/usage/python.md

```diff
@@ -3,31 +3,32 @@ title: Python Package Manager Support
 description: Python/pip dependencies support in Renovate
 ---
 
-# Python Package Manager Support
+# Python package manager support
 
 Renovate supports the following Python package managers:
 
 - `pip` (e.g. `requirements.txt`, `requirements.pip`) files
 - `pipenv` (e.g. `Pipfile`)
 - `setup.py`
 
-## Versioning Support
+## Versioning support
 
 The [PEP440](https://www.python.org/dev/peps/pep-0440/) versioning scheme has been rewritten for JavaScript for the purposes of use in this project is published as [@renovatebot/pep440](https://github.com/renovatebot/pep440).
-It supports both pinned versions as well as ranges. Legacy versions (`===` prefix) are ignored.
+It supports both pinned versions as well as ranges.
+Legacy versions (`===` prefix) are ignored.
 
-## How It Works
+## How it works
 
-1.  Renovate will search each repository for any package files it finds
-2.  Existing dependencies will be extracted from the file(s)
-3.  Renovate will look up the latest version on [PyPI](https://pypi.org/) to determine if any upgrades are available
-4.  If the source package includes a GitHub URL as its source, and has either a "changelog" file or uses GitHub releases, then Release Notes for each version will be embedded in the generated PR
+1. Renovate searches trough each repository for package files
```

Comment:
```suggestion
1. Renovate searches through each repository for package files
```

Reply:
Ah yes, a trough is for piggies to eat out of... 😄
Let's clean up this pigsty.... 🙃